pyanaconda/modules/storage/bootloader/base.py

```diff
@@ -885,6 +887,12 @@ def _set_graphical_boot_args(self):
         self.boot_args.update(args)
         self.dracut_args.update(args)
 
+    def _set_security_boot_args(self):
+        """Set LSM-related boot args."""
+        proxy = SECURITY.get_proxy()
+        if proxy.SELinux == SELinuxMode.DISABLED.value:
```

Comment:
Please, use the `SELINUX_DISABLED` constant from from `pykickstart.constants`. The implementation of the modules is considered to be private, so a module shouldn't import code of another module.